src/transforms/push.ts

```diff
@@ -143,7 +143,7 @@ export const processPush = async (github: GitHubInstallationClient, payload: Pus
 					log.info("GitHub call succeeded");
```

Comment:
You could add another log info here with `message.lenght`?

Reply:
At this point I have faith that the message length here will be what we expect, but if we are still getting a lot of grief I agree its is a good idea to throw in some debug logs around here just to confirm and rule out anything funky!